data/json/obsoletion/charge_removal.json

```diff
@@ -65,7 +73,7 @@
       "portable_game",
       "powered_earmuffs",
       "powered_earmuffs_on",
-      "pseuso_bio_picklock",
+      "pseudo_bio_picklock",
```

Comment:
This typo suggests that we ought to add some code somewhere that verifies the ids in this list are valid.

Reply:
I guess somewhe in `src/init.cpp:DynamicDataLoader::check_consistency` would be suitable. But if this list is just a temporary way to ignore charge errors with these items when loaded from an older save game, I don't foresee much need to add more to it after this PR, and suspect the whole list could be eliminated some time after 0.F.

Reply:
A couple more hours of immersion in this bug have brought me many doubts about my statement above, but either way it's not something I'm prepared to do for this PR.